External bundles

As explain here: https://github.com/substack/browserify-handbook#external-bundles
I do the following:

```
npm install jquery
browserify -r jquery --standalone jquery > jquery-bundle.js
browserify main.js --exclude jquery > bundle.js
```

index.html:

```
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>My Site</title>
  <script src="jquery-bundle.js"></script>
  <script src="bundle.js"></script>
</head>
<body>
    <h1>My Site</h1>
</body>
</html>
```

main.js:

```
var $ = require('jquery');
$(window).click(function () { document.body.bgColor = 'red' });
```

And when I open my page I get Uncaught Error: Cannot find module 'jquery'

Is it a code bug, a doc bug or am I just missing something ?

I've also tried to reproduce [the handbook's excluding example](https://github.com/substack/browserify-handbook#excluding) and ran into the same issue, using Browserify v11.0.0. However, using `--external` instead of `exclude`, as suggested by @rubennorte doesn't work either.

In both cases, you get the "Cannot find module 'jquery'" error. Any further suggestions?

--- a/readme.markdown
+++ b/readme.markdown
@@ -2025,7 +2025,7 @@ the primary bundle:
 
 ```
 $ npm install jquery
-$ browserify -r jquery --standalone jquery > jquery-bundle.js
+$ browserify -r jquery > jquery-bundle.js
 ```
 
 then we want to just `require('jquery')` in a `main.js`:

